QUEST.tsv

```diff
@@ -794,10 +794,10 @@ QUEST_20150317_000793	I wish you well… {nl}And may you and this land be blesse
 QUEST_20150317_000794	{memo X}Right. I think so too. {nl}The monsters are definitely preying on our honey. {nl}But it is better to have evidence on hand, so bring a piece of beehive {nl}from Saltus Bee Farm.
 QUEST_20150317_000795	Coward
 QUEST_20150317_000796	{memo X}I'm called a coward but I can't do anything. I'm really scared.
-QUEST_20150317_000797	You're the one the bees were whispering about. {nl}I am the Goddess of Fate, Austeja.
+QUEST_20150317_000797	$I see, you're the one the bees have been whispering about. {nl}I am the Goddess of Fate, Austeja.
 QUEST_20150317_000798	{memo X}The seal of this farmland has stabilized, thanks to you. {nl}But the other Revelators have lost themselves due to the evil force flowing from the Spring Light Woods.
-QUEST_20150317_000799	I'm using all my powers to hold the seal from opening any further… {nl}I can't move away from here, let alone save them.
-QUEST_20150317_000800	Please save them and the Spring Light Woods.
+QUEST_20150317_000799	$I will be using all my powers to keep the seal here from opening any further… {nl}Thus, I cannot move away from here, let alone save them.
+QUEST_20150317_000800	$Please save them and save the Spring Light Woods.
```

Comment:
The second save is unnecessary, them and the Spring Light Woods are both being acted upon by save.